src/es/components/atoms/breadcrumb/Breadcrumb.js

```diff
@@ -24,7 +24,7 @@ export default class Breadcrumb extends Shadow() {
    * @return {boolean}
    */
   shouldRenderCSS () {
-    return !this.root.querySelector(`:host > style[_css], ${this.tagName} > style[_css]`)
+    return !this.querySelector(`:host > style[_css], ${this.tagName} > style[_css]`)
```

Comment:
the `root` property was replaced by `this` to simplify the selector and improve performance

Reply:
the selector on this only will not work into the shadowDom. There you must use this.root

Reply:
👀 Okay. Rolled back.